spec/atom-reporter.coffee

```diff
@@ -15,8 +13,8 @@ formatStackTrace = (spec, message='', stackTrace) ->
   for line in stackTrace.split('\n')
     convertedLines.push(line) unless jasminePattern.test(line)
     break if firstJasmineLinePattern.test(line)
+  stackTrace = convertedLines.join('\n')
 
-  stackTrace = convertStackTrace(convertedLines.join('\n'), sourceMaps)
   lines = stackTrace.split('\n')
```

Comment:
Looks like the `convertedLines` are joined and then immediately split again now on the next line.